Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/Bundle.properties

```diff
@@ -240,8 +240,10 @@ AutoIngestCasePanel.bnOpen.text=&Open
 AutoIngestCasePanel.bnShowLog.toolTipText=Display case log file for selected case
 AutoIngestCasePanel.bnShowLog.text=&Show Log
 AutoIngestCasePanel.rbGroupLabel.text=Show cases accessed in the last 10:
-AutoIngestDashboard.prioritizeButton.toolTipText=Prioritizes the selected job
-AutoIngestDashboard.prioritizeButton.text=&Prioritize
 AutoIngestDashboard.refreshButton.toolTipText=Refresh displayed tables
 AutoIngestDashboard.refreshButton.text=&Refresh
 AutoIngestDashboard.jButton1.text=jButton1
+AutoIngestDashboard.prioritizeJobButton.toolTipText=Prioritizes the selected job
```

Comment:
Is there a reason why we didn't use the same tooltip text that is used in AID 1.0?